internal/config.go

```diff
@@ -15,4 +15,6 @@ var (
 	ConfigKeyGitAuthorEmail         = "git-author-email"
 	ConfigKeyPodLabelSelector       = "pod-label-selector"
 	ConfigKeyNamespaceLabelSelector = "namespace-label-selector"
+	ConfigKeyDependencaTrackBaseUrl = "dtrack-base-url"
+	ConfigKeyDependencaTrackApiKey  = "dtrack-api-key"
```

Comment:
Typo in `Dependenca` (also in the dtrack_target.go in multiple places)

Reply:
Fixed the typos.